web/controllers/student_controller.ex

```diff
@@ -43,6 +43,8 @@ defmodule Skillswheel.StudentController do
       student ->
         case Enum.member?(user_groups, student.group_id) do
           true ->
+            student = Repo.preload(student, :group)
+            IO.inspect student
```

Comment:
I'm just going to remove this line before merging. Pop a little 👍  on this comment if that's cool with you